tests/test_formato_json.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+# -*- coding: utf8 -*-
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTIBILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+
+"""Test para pyemail"""
+
+__author__ = "Mariano Reingart <reingart@gmail.com>"
+__copyright__ = "Copyright (C) 2010-2019 Mariano Reingart"
+__license__ = "GPL 3.0"
+
+
+import os
+import unittest
+import json
+import stat
+import pytest
+from decimal import Decimal
+from pyafipws.formatos.formato_json import leer, escribir
+import tempfile
+
+@pytest.mark.dontusefix
+class TestFormatoJSON(unittest.TestCase):
```

Comment:
@ABHISHEKSONI121 Could you write the test with pytest? You could use the pytest `tmp_path` fixture for this.

Reply:
sorry for that
I have updated the tests and also ran black and flake8 before committing as you asked